### PR TITLE
Add AEAP info dialog & strings

### PR DESCRIPTION
**Pull request by Hocuri**

### CODE REVIEW

File: res/values/strings.xml [hunk 1]
```diff
@@ -446,6 +446,8 @@
     <string name="tab_webxdc_empty_hint">Webxdc content shared in this chat will appear here.</string>
     <string name="media_preview">Media Preview</string>
     <string name="send_message">Send Message</string>
+    <string name="aeap_addr_changed">%1$s changed their address from %2$s to %3$s</string>
+    <string name="aeap_explanation">You changed your email address.\n\nContacts using Delta Chat will automatically replace you in the groups you send a message to.\n\nIf possible, set up your old email provider to forward all emails to your new email address.</string>
```
Comment (r10s) [inline res/values/strings.xml:450]:
```suggestion
    <!-- the explanation is shown (1) as a modal dialog with the buttons "Cancel" and "Continue" as well as (2) as a device message -->
    <string name="aeap_explanation">You changed your email address.\n\nContacts using Delta Chat will automatically replace you in the groups you send a message to.\n\nIf possible, set up your old email provider to forward all emails to your new email address.</string>
```

File: res/values/strings.xml [hunk 2]
```diff
@@ -446,6 +446,8 @@
     <string name="tab_webxdc_empty_hint">Webxdc content shared in this chat will appear here.</string>
     <string name="media_preview">Media Preview</string>
     <string name="send_message">Send Message</string>
+    <string name="aeap_addr_changed">%1$s changed their address from %2$s to %3$s</string>
```
Comment (r10s) [inline res/values/strings.xml:449]:
```suggestion
    <!-- Placeholder %1$s will be replaced by the name of the contact changing their address. Placeholders %2$s and %3$s will be replaced by old/new email addresses. -->
    <string name="aeap_addr_changed">%1$s changed their address from %2$s to %3$s</string>
```

File: src/org/thoughtcrime/securesms/connect/DcHelper.java [hunk 3]
```diff
@@ -207,6 +212,8 @@ public static void setStockTranslations(Context context) {
     // cmp. https://github.com/deltachat/deltachat-android/issues/2187
     dcContext.setStockTranslation(120, context.getString(R.string.qrshow_join_group_hint).replace("\"", ""));
     dcContext.setStockTranslation(121, context.getString(R.string.connectivity_not_connected));
+    dcContext.setStockTranslation(122, context.getString(R.string.aeap_addr_changed));
+    dcContext.setStockTranslation(123, context.getString(R.string.aeap_explanation));
```
[the remark this answers is not in the document]
Comment (Hocuri) [reply src/org/thoughtcrime/securesms/connect/DcHelper.java:216]: What do we do with the fallback then?

https://github.com/deltachat/deltachat-core-rust/blob/56699a29d29d2c058a8b4244dd34b7b4a96c711c/src/stock_str.rs#L340

If we want to do it completely correctly, we need to add the link there, too. If we don't do this, UIs that didn't add this string will not show the link at all. Then again, maybe that's fine.

FWIW, the link was intended to already be the final link, though I just noticed it's not correct, we need to change this (https://delta.chat/550/en/2022-07-10-aeap -> https://delta.chat/en/2022-07-10-aeap).

[the remark this answers is not in the document]
Comment (r10s) [reply src/org/thoughtcrime/securesms/connect/DcHelper.java:216]: > What do we do with the fallback then?

just do not add the link there :) 

there is no need to add the link to the fallback. the fallback is not really shown in any of the main uis and are a bit different anyway, often also shorter.

**Review by r10s — APPROVED** [review]
lgtm, not build or test.

**Pull request merged**